lib/broccoli/fastboot-config.js

```diff
@@ -104,7 +104,9 @@ FastBootConfig.prototype.readAssetManifest = function() {
     var assetMap = JSON.parse(fs.readFileSync(assetMapPath));
     return assetMap;
   } catch (e) {
-    // No asset map was found, proceed as usual
+    if (assetMapPath) {
```

Comment:
IMO, you should only emit warnings if they are actionable. In this case, it will always emit a warning in development mode that you can't get rid of, because asset rewriting is not turned on in development mode. That would drive me crazy, and it will make people less likely to see other, *real* warnings that we might emit in the future.